src/storybook.js

```diff
@@ -155,6 +155,46 @@ var staticServer = exports.staticServer = function(config, options, callback) {
   }).catch(callback);
 };
 
+var setStorybookConfig = exports.setStorybookConfig = function(storybookApp, storybookVersion, storybookConfigDir) {
+  var configPath = path.resolve(process.cwd(), storybookConfigDir, 'config.js');
+  var isNewFile = false;
+  if (!fs.existsSync(configPath)) {
+    // handle declaritive configuration and preview.js in Storybook 5.3+
+    if (storybookVersion.major >= 5 && semver.gt(storybookVersion.full, '5.2.0')) {
+      configPath = path.resolve(process.cwd(), storybookConfigDir, 'preview.js');
+      if (!fs.existsSync(configPath)) {
+        fs.writeFileSync(configPath, '', 'utf8');
+        isNewFile = true;
+      }
+    } else {
+      throw new Error('Storybook config file not found: ' + configPath);
+    }
+  }
+  var configBody = fs.readFileSync(configPath, 'utf8');
+  var templateType = 'default';
+  if (storybookVersion.major === 2) {
+    templateType = 'v' + storybookVersion.major;
+  }
+  var codeTemplate = fs.readFileSync(__dirname + '/templates/' + templateType + '.template', 'utf8');
```

Comment:
@loyalchow what do you think of using `path.resolve`(or `parth.join`) for  `__dirname + '/templates/' + templateType + '.template'` ?